src/sonic-py-common/sonic_py_common/device_info.py

```diff
@@ -43,7 +43,7 @@
 
 def get_localhost_info(field):
```

Comment:
>get_localhost_info [](http://example.com/codeflow?start=4&length=18)

If you intention is to fix hostcfgd, you do not need to modify this file.

Reply:
No, this is not the main intension. Please see the following back trace:

```
Traceback (most recent call last):', '  File "/usr/local/bin/sonic-installer", line 5, in <module>, 
    from sonic_installer.main import sonic_installer', '  File "/usr/local/lib/python3.9/dist-packages/sonic_installer/main.py", line 7, in <module>', 
    import utilities_common.cli as clicommon', '  File "/usr/local/lib/python3.9/dist-packages/utilities_common/cli.py", line 189, in <module>',
     iface_alias_converter = InterfaceAliasConverter()', '  File "/usr/local/lib/python3.9/dist-packages/utilities_common/cli.py", line 126, in __init__', 
    self.port_dict = multi_asic.get_port_table()', '  File "/usr/local/lib/python3.9/dist-packages/sonic_py_common/multi_asic.py", line 301, in get_port_table',
     ports = get_port_table_for_asic(ns)', '  File "/usr/local/lib/python3.9/dist-packages/sonic_py_common/multi_asic.py", line 315, in get_port_table_for_asic',
     config_db = connect_config_db_for_ns(namespace)', '  File "/usr/local/lib/python3.9/dist-packages/sonic_py_common/multi_asic.py", line 47, in connect_config_db_for_ns',
     config_db.connect()',
   File "/usr/lib/python3/dist-packages/swsscommon/swsscommon.py", line 1829, in connect', 
   return _swsscommon.ConfigDBConnector_Native_connect(self, wait_for_init, retry_on)', 
'RuntimeError: Unable to connect to redis: Cannot assign requested address
```